components/forms/change-password/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useRef } from 'react'
+
+import styles from './styles.css'
+
+import { Card } from 'design'
+import { withGlobalStore } from 'store'
+
+const { IDP_URL } = process.env
+
+const ChangePassword = ({ className, store, tag = 'section' }) => {
+  const iframeRef = useRef(null)
+  const searchParams = {
+    identity_provider_url: IDP_URL,
+    email: store.user?.email,
+  }
+
+  // remove empty params
+  Object.keys(searchParams).forEach(
+    (key) => searchParams[key] === undefined && delete searchParams[key]
+  )
+
+  const search = new URLSearchParams(searchParams).toString()
+  const resize = () => {
+    iframeRef.current.style.height = `${iframeRef.current.contentWindow.document.body.offsetHeight}px`
+  }
+
+  return (
+    <Card className={className} tag={tag}>
+      <h2>Change password</h2>
+      <iframe
+        ref={iframeRef}
+        className={styles.changePasswordIframe}
+        title="Password change form"
+        src={`/secure/reset.html?${search}`}
+        scrolling="no"
+        allowtransparency="true"
```

Comment:
Why is this needed? It's not standard but the default behaviour is that the iframe is transparent.